src/v0/util/index.test.js

```diff
@@ -985,3 +987,65 @@ describe('removeHyphens', () => {
     });
   });
 });
+
+describe('convertToUuid', () => {
+  const NAMESPACE = v5.DNS;
+
+  test('should generate UUID for valid string input', () => {
+    const input = 'testInput';
+    const expectedUuid = v5(input, NAMESPACE);
+    const result = convertToUuid(input);
+    expect(result).toBe(expectedUuid);
+  });
+
+  test('should generate UUID for valid numeric input', () => {
+    const input = 123456;
+    const expectedUuid = v5(String(input), NAMESPACE);
+    const result = convertToUuid(input);
+    expect(result).toBe(expectedUuid);
+  });
+
+  test('should trim spaces and generate UUID', () => {
+    const input = '   testInput   ';
+    const expectedUuid = v5('testInput', NAMESPACE);
+    const result = convertToUuid(input);
+    expect(result).toBe(expectedUuid);
+  });
+
+  test('should throw an error for empty input', () => {
+    const input = '';
+    expect(() => convertToUuid(input)).toThrow(InstrumentationError);
+    expect(() => convertToUuid(input)).toThrow('Input is empty or invalid.');
+  });
+
+  test('does not throw an error for null input', () => {
```

Comment:
for undefined & null inputs, it should throw error

Reply:
done